lib/jsonDir.js

```diff
@@ -22,9 +22,12 @@ var collectJSONS = function(options) {
             jsonOutput.push(json);
         }
 
-        if (!cucumberJson || typeof cucumberJson[0] === 'undefined') {
+        if (!cucumberJson || typeof cucumberJson[0] === 'undefined' && !options.ignoreBadJsonFile) {
```

Comment:
Can you add parenthesis to the both _if & else_ conditions?

```
if ((!cucumberJson || typeof cucumberJson[0] === 'undefined') && !options.ignoreBadJsonFile) 
```

Because it will throw Error even if `options.ignoreBadJsonFile = false` but if `cucumberJson` isn't valid object.